### PR TITLE
use adapters to get the relationship direction

### PR DESCRIPTION
**Pull request by MacondoExpress**
Support query direction

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the Neo4j GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-3770.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/src/schema-model/relationship/model-adapters/RelationshipAdapter.ts [hunk 1]
```diff
@@ -97,6 +97,38 @@ export class RelationshipAdapter {
     public findAttribute(name: string): AttributeAdapter | undefined {
         return this.attributes.get(name);
     }
+    /**
+     * translation-only
+     *
+     * @param directed the direction asked during the query, for instance "friends(directed: true)"
+     * @returns the direction to use in the CypherBuilder
+     **/
+    public getCypherDirection(directed?: boolean): "left" | "right" | "undirected" {
```
Comment (angrykoala) [inline packages/graphql/src/schema-model/relationship/model-adapters/RelationshipAdapter.ts:106]: A thing I don't like about this approach is that we are coupling our internal model (schema model) to the cypher-builder API

It is not a huge problem, as the API is still somewhat agnostic and we own both pieces, but it feels like this is the wrong place for this (at least while the adaptor are so coupled with the schema model)

**Comment by angrykoala** [comment]
Is this worth it at the moment? I'm not so sure about coupling the cypher logic to the model tbh

Also, a couple of small comments on the logic to generate these values

File: packages/graphql/src/schema-model/relationship/model-adapters/RelationshipAdapter.ts (hunk 1 above)
Comment (MacondoExpress) [reply packages/graphql/src/schema-model/relationship/model-adapters/RelationshipAdapter.ts:106]: I agree, but the plan was to keep these adapters in the schema-model folder only during the refactoring, but then move them outside in a specialized fashion. Happy to delete these adapaters if the transition will not work out.

**Pull request merged**